core/pom.xml

```diff
@@ -548,6 +548,12 @@
       <groupId>org.owasp.esapi</groupId>
       <artifactId>esapi</artifactId>
       <version>2.4.0.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
```

Comment:
This exclusion makes sense to me it excludes a transitive dependency on log4j 1.2 implementation.